src/model/user.rs

```diff
@@ -458,6 +458,19 @@ impl Hash for User {
     }
 }
 
+impl Default for User {
+    fn default() -> Self {
```

Comment:
Documentation should be added to inform users which default values this method assumes for a `User`.

e.g.:
```rust
/// Initializes a `User` with default values. Setting the following:
/// - **id** to `UserId(210)`
/// - **avatar** to `Some("abc")`
/// ....
```